docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.general.scss

```diff
@@ -1,13 +1,18 @@
 .hb-page-width {
-  @include hb-page-width;
```

Comment:
I dont think this mixin makes sense for our styles.. what do others think? I think Merani had wanted 3rem to kinda pad everything in our design

Reply:
I think the goal was to use the Decanter mixin but if that isn't going to work for our layouts we can abandon it.